### PR TITLE
Update UML diagrams

### PR DESCRIPTION
**Pull request by JacobBredin**
Add newest version of UML diagrams to each package and add missing readme files

### CODE REVIEW

File: src/main/java/com/thebois/views/game/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# com.thebois.views.game
+# com.thebois.views.gameviews
 
 ## UML Class Diagram
 
-![com.thebois.views.game](../../../../../../../documents/diagrams/com.thebois.views.game.jpg "com.thebois.views.game")
+![com.thebois.views.gameviews](./../../../../../../../documents/diagrams/com.thebois.views.gameviews.jpg "com.thebois.views.gameviews")
```
Comment (Matte02) [inline src/main/java/com/thebois/views/game/README.md:5]: Should be "game" and not "gameviews"

File: src/main/java/com/thebois/views/info/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# com.thebois.views.info
+# com.thebois.views.infoviews
 
 ## UML Class Diagram
 
-![com.thebois.views.info](../../../../../../../documents/diagrams/com.thebois.views.info.jpg "com.thebois.views.info")
+![com.thebois.views.infoviews](./../../../../../../../documents/diagrams/com.thebois.views.infoviews.jpg "com.thebois.views.infoviews")
```
Comment (Matte02) [inline src/main/java/com/thebois/views/info/README.md:5]: Same as above, swap to "info" from "infoview"

**Comment by martinjonsson01** [comment]
In the "files changed" tab it says you've added two files:
image

**Review by martinjonsson01 — APPROVED** [review]
Ah, I see. You never modified them that's why I can't see them.

Looks good now!

**Review by Matte02 — APPROVED** [review]
Well done!

**Pull request merged**